starcheck/src/starcheck.pl

```diff
@@ -49,21 +49,43 @@
 use Inline Python => q{
 
 import os
+import six
 from chandra_aca.star_probs import set_acq_model_ms_filter
 import starcheck
 from starcheck.pcad_att_check import make_pcad_attitude_check_report, check_characteristics_date
 from starcheck.calc_ccd_temps import get_ccd_temps
 from starcheck.version import version
 
+def debyte_kwargs(kwargs):
+    if six.PY2:
+       return kwargs
+    else:
+       td = dict()
+       for key, val in six.iteritems(kwargs):
+           try:
```

Comment:
This touches on a philosophical question of duck-typing: for any object that has a `decode()` method, is it the right thing to call it?  For this case I'd rather take an opt-in approach and handle the case we know about, which is `bytes`:
```
if isinstance(key, bytes):
    key = key.decode('ascii')
```
And make the encoding explicit for this use case, since otherwise it uses the system default which is not necessarily predictable.

Reply:
BTW, an alternate one-line hack that I think works for arbitrary pure Python structures is:
```
In [11]: r = {b'name': {b'joe': [1, 'jim', b'joe']}}

In [12]: repr(r)
Out[12]: "{b'name': {b'joe': [1, 'jim', b'joe']}}"

In [13]: ast.literal_eval(re.sub(r"b'", "'", repr(r)))
Out[13]: {'name': {'joe': [1, 'jim', 'joe']}}
```
This does assume that the string `b'` never occurs naturally within a string, but I think that is the case in this application.

Reply:
I suppose I should also do more checking to confirm that I have no strings that end in `b'` as you note:
```
In [7]: r = {b'name': {b'joe': [1, 'jim', b'bob']}}

In [8]: ast.literal_eval(re.sub(r"b'", "'", repr(r)))
Out[8]: {'name': {'joe': [1, 'jim', 'bo']}}
```